brightray/browser/browser_main_parts.cc

```diff
@@ -252,11 +245,35 @@ void BrowserMainParts::PreMainMessageLoopStart() {
 }
 
 void BrowserMainParts::PreMainMessageLoopRun() {
+  auto* command_line = base::CommandLine::ForCurrentProcess();
+
+  // TODO(poiru): Link base_switches.cc and use
+  // switches::k{Enable,Disable}Features instead.
+  const auto enable_features =
+      command_line->GetSwitchValueASCII("enable-features");
+  auto disable_features = command_line->GetSwitchValueASCII("disable-features");
```

Comment:
Can you ship and link `base_static` from libcc for linux and macOS to fix this, we already build it.

Reply:
Opened the libcc PR. I'm happy to do this as a follow-up or wait for the libcc builds to complete.